app/overrides/spree/admin/shared/_configuration_menu/add_stripe_connect_settings.html.haml.deface

```diff
@@ -1,4 +1,4 @@
 // insert_bottom "[data-hook='admin_configurations_sidebar_menu']"
 
 %li
-  = link_to t('.stripe_connect'), main_app.edit_admin_stripe_connect_settings_path
+  = link_to t('admin.stripe_connect_settings.edit.title'), main_app.edit_admin_stripe_connect_settings_path
```

Comment:
All the translations of settings are using lazy lookup from admin._settings_.edit, the only absolute reference is on this menu deface file (and only for this stripe connect settings). If we don't do this, the relative reference would leave an isolated entry on the en.yml file, which would just add noise to it (see removed line 2599 in en.yml below).